modules/tools/vehicle_calibration/sensor_calibration/extract_data.py

```diff
@@ -343,29 +304,33 @@ def extract_data(record_file, output_path, channel_list,
         if channel in SMALL_TOPICS:
             channel_msgs_dict[channel] = list()
 
-    record_reader = RecordReader(record_file)
-    for msg in record_reader.read_messages():
-        if msg.topic in channel_list:
-            # Only care about messages in certain time intervals
-            msg_timestamp_sec = msg.timestamp / 1e9
-            if not in_range(msg_timestamp_sec, start_timestamp, end_timestamp):
-                continue
-
-            channel_occur_time[msg.topic] += 1
-            # Extract the topic according to extraction_rate
-            if channel_occur_time[msg.topic] % extraction_rate_dict[msg.topic] != 0:
-                continue
-
-            ret, channel_msgs_dict = extract_channel_data(channel_output_path[msg.topic], msg,
-                                       channel_msgs_dict)
-            # Calculate parsing statistics
-            if ret is False:
-                process_msg_failure_num += 1
-                if channel_success_dict[msg.topic] is True:
-                    channel_success_dict[msg.topic] = False
-                    process_channel_failure_num += 1
-                    process_channel_success_num -= 1
-                    print('Failed to extract data from channel: %s' % msg.topic)
+    for record_file in record_file_list:
+        record_reader = RecordReader(record_file)
+        for msg in record_reader.read_messages():
+            if msg.topic in channel_list:
+                # Only care about messages in certain time intervals
+                msg_timestamp_sec = msg.timestamp / 1e9
+                if not in_range(msg_timestamp_sec, start_timestamp, end_timestamp):
+                    continue
+
+                channel_occur_time[msg.topic] += 1
+                # Extract the topic according to extraction_rate
+                if channel_occur_time[msg.topic] % extraction_rate_dict[msg.topic] != 0:
+                    continue
+
+                ret, channel_msgs_dict =\
+                    extract_channel_data(channel_output_path[msg.topic],
+                                         msg, channel_msgs_dict)
+
+                # Calculate parsing statistics
+                if ret is False:
+                    process_msg_failure_num += 1
+                    if channel_success_dict[msg.topic] is True:
+                        channel_success_dict[msg.topic] = False
```

Comment:
Remove `is True`, and change all `XXX is False` to `not XXX`

Reply:
done. thanks